src/components/Table/GridTable.stories.tsx

```diff
@@ -199,6 +203,323 @@ export function NestedRows() {
   );
 }
 
+export function NestedCards() {
+  const nameColumn: GridColumn<NestedRow> = {
+    header: () => "Name",
+    parent: (row) => ({
+      content: <div css={Css.base.$}>{row.name}</div>,
+      value: row.name,
+    }),
+    child: (row) => ({
+      content: <div css={Css.sm.$}>{row.name}</div>,
+      value: row.name,
+    }),
+    grandChild: (row) => ({
+      content: <div css={Css.xs.$}>{row.name}</div>,
+      value: row.name,
+    }),
+  };
+  const spacing = { brPx: 4, pxPx: 8, spacerPx: 8 };
+  const nestedStyle: GridStyle = {
+    nestedCards: {
```

Comment:
`GridStyle` learned a new `nestedCards` that:

a) takes the background color and optional border color of each card (based on kind)
b) enables within GridTable creation of the nested card DOM stuff (i.e. no other flag / `GridTable` prop is necessary, we can just look at the style being used and do the right thing)

Reply:
**thought:** Maybe GridTable should provide a default `nestedStyle`, probably would be good to just default to whatever we're going to need first. Then it'd also be a good reference for developers to see how it should/can be set up.